ui/src/app/cohort-search/node-info/node-info.component.ts

```diff
@@ -118,6 +118,7 @@ export class NodeInfoComponent implements OnInit, OnDestroy, AfterViewInit {
      * fire a request for children (if there are any)
      */
     event.stopPropagation();
+    console.log(this.node.toJS());
```

Comment:
Nit: Remove console log statement if not needed?

Reply:
Not needed, removed.